README.md

```diff
@@ -6,10 +6,10 @@ This action for [Changesets](https://github.com/atlassian/changesets) creates a
 
 ### Without Publishing
 
-Create a file at `.github/workflows/release.yml` with the following content.
+Create a file at `.github/workflows/version.yml` with the following content.
 
 ```yml
-name: Release
+name: Version
```

Comment:
My thoughts were release is the right word because it's the combination of version + sometimes build + publish (https://github.com/atlassian/changesets/pull/158) any objections to that?

Reply:
I changed the one that was specifically 'copy this if you just want versioning' as opposed to the 'this one does versioning and commit' - to try and make the 'version' vs 'release' definition more consistent. Happy to move it back.

Reply:
Hmmm, that makes sense. My thoughts on naming the things release was that I was planning on renaming this action to `changesets/release-action` so having some of the things named version and some release would be strange.

Maybe we could have a version action and a publish action but separating the things because of naming seems like not the best idea?

Reply:
I think sticking with `Release` is simply going to be easier (I don't want to make this two actions if we can help it)

---

meta-discussion: Is it better to have one `changesets` GH action that's configurable with multiple actions, vs having different things for different tasks. The way this was initially set up, I thought we were going to be getting one package, but we've since divided. What's the reasons?